build/docker/dos-service-fuzzy-search-api/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM 461183108257.dkr.ecr.eu-west-2.amazonaws.com/uec/service-finder/api-base:1.0.20191119
+
+ADD assets/application/dos-service-fuzzy-search-api.jar /application/
+ENTRYPOINT ["java", "-jar", "/application/dos-service-fuzzy-search-api.jar"]
```

Comment:
If we drop the dependency on the closed-source image we need convert `ENTRYPOINT` to `CMD`.

PS. I know that all of that is temporary and we are going to move to Lambdas but thought it would be good practice to do it the right way and check if all of that actually works shall we need it later.

Reply:
Done.